src/controllers/auth/logout.ts

```diff
@@ -0,0 +1,10 @@
+import { Express } from '../../types';
+
+export const logout: Express.MiddleWare = async (req, res) => {
+  res.cookie('jwt', '', { expires: new Date(0), httpOnly: true });
```

Comment:
what's the difference between `new Date(0)` and `new Date()`? can we use `new Date()`?

Reply:
`new Date(0)` creates a Date object that represents a specific moment in the past, while `new Date()` creates a Date object that represents the current moment in time. So, we can't use `new Date()` here.

Reply:
i don't think that really matters since by the time the user attempts to log back in, the token will definitely already be expired - i think we should opt to use `new Date()` unless you can prove that changing the data will break your logic